### PR TITLE
[DDW-1069] Add linking steps to external packages in readme

### PR DESCRIPTION
**Pull request by shawnbusuttil**
Adding linking steps with external libraries such as `react-polymorph`.

## Screenshots

No UI changes.

## Testing Checklist

No QA needed.

### Basics

- [ ] PR assigned to the PR author(s)
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [ ] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [ ] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [ ] PR link is added to a Jira ticket, ticket moved to In Review
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR contains screenshots (in case of UI changes)
- [ ] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by lucas-barros — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -224,6 +224,25 @@ You can debug the main process by following one of these approaches:
...
 
+### Linking with UI Libraries (e.g. React Polymorph)
+
+You can link libraries with Daedalus using one of the following steps:
+
+#### 1) Using `yalc`
+
+1) Install `yalc` globally using `yarn global add yalc`.
+2) Run `yalc publish` from the library's root directory that you want to link with Daedalus.
+3) Switch to Daedalus and run `yalc add <package-name>` or `yalc link <package-name>`.
```
Comment (przemyslaw-wlodek) [inline README.md:235]: This command creates `.yalc` dir. Should we also consider updating our `.gitignore` if we recommend that way of linking?

**Review by przemyslaw-wlodek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shawnbusuttil) [reply README.md:235]: Yes

Comment (przemyslaw-wlodek) [inline README.md:235]: Let's use `yalc link` only as it doesn't modify the `package.json` which could be accidentally committed.

File: README.md [hunk 2]
```diff
@@ -224,6 +224,25 @@ You can debug the main process by following one of these approaches:
...
 The inspector runs on port 9229
 
+### Linking with UI Libraries (e.g. React Polymorph)
+
+You can link libraries with Daedalus using one of the following steps:
+
+#### 1) Using `yalc`
+
+1) Install `yalc` globally using `yarn global add yalc`.
+2) Run `yalc publish` from the library's root directory that you want to link with Daedalus.
```
Comment (przemyslaw-wlodek) [inline README.md:234]: I'd suggest to use a `yalc push` as otherwise `publish` requires a `--push` flag in order to have updates in Daedalus.

**Review by przemyslaw-wlodek — CHANGES_REQUESTED** [review]

**Review by renanvalentin — APPROVED** [review]

**Review by przemyslaw-wlodek — APPROVED** [review]

**Review by danielmain — APPROVED** [review]

**Pull request merged**